### PR TITLE
Fixed: Eyedropper experiment not working on Chromium based browsers (ED-2960)

### PR DESCRIPTION
**Pull request by StyleShit**

### CODE REVIEW

File: modules/elements-color-picker/assets/js/editor/utils.js [hunk 1]
```diff
@@ -10,7 +10,7 @@
 export const addNamespaceHandler = ( elements, nsEvent, callback, options = {} ) => {
 	const [ event ] = nsEvent.split( '.' );
 
-	if ( ! ( elements instanceof NodeList ) ) {
+	if ( ! ( Symbol.iterator in Object( elements ) ) ) {
```
Comment (matipojo) [inline modules/elements-color-picker/assets/js/editor/utils.js:13]: Explain

File: modules/elements-color-picker/assets/js/editor/utils.js [hunk 2]
```diff
@@ -33,7 +33,7 @@ export const addNamespaceHandler = ( elements, nsEvent, callback, options = {} )
 export const removeNamespaceHandler = ( elements, nsEvent ) => {
 	const [ event ] = nsEvent.split( '.' );
 
-	if ( ! ( elements instanceof NodeList ) ) {
+	if ( ! ( Symbol.iterator in Object( elements ) ) ) {
```
Comment (matipojo) [inline modules/elements-color-picker/assets/js/editor/utils.js:36]: Explain

**Review by matipojo — CHANGES_REQUESTED** [review]

File: modules/elements-color-picker/assets/js/editor/utils.js (hunk 2 above)
Comment (StyleShit) [reply modules/elements-color-picker/assets/js/editor/utils.js:36]: Done.

File: modules/elements-color-picker/assets/js/editor/utils.js (hunk 1 above)
Comment (StyleShit) [reply modules/elements-color-picker/assets/js/editor/utils.js:13]: Done.

**Review by matipojo — CHANGES_REQUESTED** [review]
Please add a temp fix for the `global is undefined` Qunit error,

**Pull request merged**